client/ayon_core/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -172,4 +198,10 @@ def get_instance_attr_defs(self):
 
     def get_pre_create_attr_defs(self):
         # Use same attributes as for instance attributes
-        return self.get_instance_attr_defs()
+        selection_list = []
+        if  substance_painter.application.version_info()[0] >= 10:
+            selection_list = [
+                BoolDef("use_selection", label="Use selection",
+                        tooltip="Select Layer Stack(s) for exporting")
+            ]
```

Comment:
I like this more than setting `selection_list` as an empty list? Also, it's not a selection list? :)
```suggestion
        attr_defs = []
        if  substance_painter.application.version_info()[0] >= 10:
            attr_defs.append(
                BoolDef("use_selection", label="Use selection",
                        tooltip="Select Layer Stack(s) for exporting")
            )
```